packages/noya-state/src/selectors.ts

```diff
@@ -125,6 +133,16 @@ export const getSelectedLayersWithFixedRadius = (
   ).filter((layer): layer is Sketch.Rectangle => layer._class === 'rectangle');
 };
 
+export const getSelectedColorSwatches = (
+  state: ApplicationState,
+): Sketch.Swatch[] => {
+  const sharedSwatches = getSharedSwatches(state);
+
+  return sharedSwatches.filter((swatch) =>
+      state.selectedSwatchIds.includes(swatch.do_objectID),
+    ).filter(swatch => swatch.value._class === 'color');
```

Comment:
I don't think we need to filter on `_class`. The Sketch types are weird and I think the other shared styles aren't in the swatch object